experimental/generation/generator/packages/library/templates/professional/config.json.lg

```diff
@@ -11,7 +11,7 @@ ${join(foreach(locales, locale, localeTemplate(locale)), ','+EOL())}
 ```
 
 # localeTemplate(locale)
--```    "./${knowledgeDir()}${locale}/${prefix}.${locale}.lu": {
+-```    "${prefix}.${locale}": {
```

Comment:
>${prefix} [](start = 9, length = 9)

Why did you change this?  For composer this is where the results need to be.

Reply:
I asked Fei, he said in Composer, the config file for cross train will be generated automatically.  Could we keep the change?